File: LeetCode题目集/58. Length of Last Word(easy).java
/**
 * @Date    : 2019-01-23 21:13:34
 * @Author  : 酸饺子 (changzheng300@foxmail.com)
 * @Link    : https://github.com/SourDumplings
 * @Version : $Id$
 *
 * https://leetcode.com/problems/length-of-last-word/
*/

class Solution
{
    public int lengthOfLastWord(String s)
    {
        String[] res = s.trim().split(" ");
        if (res.length == 0)
        {
            return 0;
        }
        else
            return res[res.length-1].length();
    }
}

File: LeetCode题目集/58. Length of Last Word(easy).cpp
/*
 @Date    : 2019-01-23 21:04:44
 @Author  : 酸饺子 (changzheng300@foxmail.com)
 @Link    : https://github.com/SourDumplings
 @Version : $Id$
*/

/*
https://leetcode.com/problems/length-of-last-word/
 */

class Solution
{
public:
    int lengthOfLastWord(string s)
    {
        istringstream is(s);
        string str;
        while (is >> str);
        return str.length();
    }
};
